inst/css/filter-panel.css

```
@@ -267,11 +279,16 @@ a.remove_all:hover {
   border-bottom-right-radius: 0;
 }
 
-.filter-card-header:hover,
-.filter-card-body:hover {
+.filter-card-header:hover {
   background-color: var(--bs-gray-200, #e9ecef);
 }
 
+.filter-card-header[aria-expanded="true"]:hover {
+  transition-property: background-color;
+  transition-duration: 150ms;
+  background-color: #f5f5f5;
```

```suggestion
  background-color: #eaecef;
```
It would be nice to use the same background that we use for the active hover of the card header, regardless of whether the card is expanded or not.

https://github.com/insightsengineering/teal.slice/assets/49812166/f06fbe3f-2a9a-4858-b902-bc879cbd4af3

---

With the default hover-darkening the contrast between the highlighted card body and the darkened card header is jarring.